bin/node-template/runtime/src/lib.rs

```diff
@@ -197,8 +197,13 @@ impl frame_system::Config for Runtime {
 
 impl pallet_randomness_collective_flip::Config for Runtime {}
 
+parameter_types! {
+	pub const MaxAuthorities: u32 = 10;
```

Comment:
should be a bit higher, no? 10 is likely too much for folks playing around with node template.

Reply:
Sorry, too high or too low? Your first sentence says too high but your second says "it's too much"...

Reply:
fixed.

Reply:
```suggestion
	pub const MaxAuthorities: u32 = 32;
```